library/query_self_hosted_workflow.yml

```diff
@@ -0,0 +1,25 @@
+id: RP-10
+
+info:
+  name: Self Hosted Run
+  severity: high
+  description: Self Hosted runners should not be used for public repositories.
+  full-description: |
+    Self Hosted runners do not have the same security controls as GitHub Hosted runners and
+    do not have the guarantees of a clean ephemeral environment. Self Hosted runners should
+    only be used for private repositories.
```

Comment:
Note: we need to check somehow if a project is public or private
for private organization scanning we scan everything, no matter if private or public.
Maybe we can create an issue about that

Reply:
https://github.com/CycodeLabs/raven/issues/112